public/css/home.css

```diff
@@ -606,7 +611,7 @@ h2 {
     background-color: #000;
 }
 
-.registryOptionText {
+.registry-option-text {
```

Comment:
why was this class name changed?

Reply:
Cause on the home blade, it is written like that. I'm guessing it got updated with lint, to keep standardization of class names?